From 2a4e295a7740040799673204367a2dece0e6a699 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 9 Dec 2020 11:43:56 +0800
Subject: [PATCH] arm64: dts: rockchip: Remove rk3568-android.dtsi from
 rk3568-evb1-ddr4-v10.dtsi

rk3568-android.dtsi should included in dts.

Signed-off-by: Tao Huang <huangtao@rock-chips.com>
Change-Id: I08ee2e2930152f74c726359e9b8880122255dd80
---
 arch/arm64/boot/dts/rockchip/rk3568-evb1-ddr4-v10.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3568-evb1-ddr4-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3568-evb1-ddr4-v10.dtsi
index 54f50d3ac2c6..1e9c81fc0832 100644
--- a/arch/arm64/boot/dts/rockchip/rk3568-evb1-ddr4-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3568-evb1-ddr4-v10.dtsi
@@ -10,7 +10,6 @@
 #include <dt-bindings/pinctrl/rockchip.h>
 #include "rk3568.dtsi"
 #include "rk3568-evb.dtsi"
-#include "rk3568-android.dtsi"
 
 / {
 	model = "Rockchip RK3568 EVB1 DDR4 V10 Board";
-- 
2.35.3

